test-parachains/adder/build.rs

```diff
@@ -16,10 +16,16 @@
 
 use wasm_builder_runner::{build_current_project_with_rustflags, WasmBuilderSource};
 
+use std::env;
+
 fn main() {
 	build_current_project_with_rustflags(
 		"wasm_binary.rs",
-		WasmBuilderSource::Crates("1.0.5"),
+		WasmBuilderSource::Crates("1.0.7"),
 		"-C link-arg=--import-memory",
 	);
+
+	if env::var("TARGET") == Ok("wasm32-unknown-unknown".into()) {
+		println!("cargo:rustc-cfg=no_std");
```

Comment:
I'm not sure I understand why this is needed. didn't you remove the no_std feature ?

Reply:
Ahhh :( Some leftovers.